app/controllers/api/v1/application_controller.rb

```diff
@@ -51,7 +51,7 @@ def check_token_and_set_application
 
       def authenticate
         api_user = authenticate_or_request_with_http_token do |token, _options|
-          ApiUser.find_by(token:)
+          ApiUser.find_by(token: "7f3ac902-aa13-4148-9b36-2f9ac56bcccc")
```

Comment:
Could we give this token a variable name perhaps? Otherwise it's easy to lose track of which one is which.